README.md

```diff
@@ -342,6 +342,13 @@ Note that your GCP identity is case sensitive but `gcloud info` as of Google Clo
 
 After running the above, if you see `Clusterrolebinding "cluster-admin-binding" created`, then you are able to continue with the setup of this service.
 
+#### Probes
```

Comment:
```suggestion
#### Health endpoints
```

Reply:
I'd prefer the former since per-port endpoints are not consistent across the binary and its telemetry expositions. Also, we don't have a dedicated `/readyz` but the `readinessProbe` makes use of the telemetry port's `/metrics` to determine if we are able to serve requests.

Reply:
Friendly ping. :)

Reply:
Added a section for endpoints as well, PTAL.

Reply:
Should we remove this paragraph? I don't think it's specific to kube-state-metrics and having the endpoints documented is good enough. :)

Reply:
Done, PTAL.